src/Microsoft.OData.Client/DeepInsertSaveResult.cs

```diff
@@ -118,6 +120,65 @@ internal void DeepInsertRequest<T>(T resource)
             }
         }
 
+        /// <summary>
+        /// Asynchronous deep insert request.

```

Comment:
Asynchronous ?

Reply:
This method initiates an asynchronous call that returns an `IAsyncResult`.